apps/smart-contracts/core/test/WithdrawHook.test.ts

```diff
@@ -134,6 +144,157 @@ describe('=> WithdrawHook', () => {
 
       expect(depositRecord.recordWithdrawal).to.be.calledWith(user.address, TEST_AMOUNT_AFTER_FEE)
     })
+
+    describe('global withdraw limit testing', () => {
+      it('sets last global reset to current time if 0', async () => {
+        expect(await withdrawHook.getLastGlobalPeriodReset()).to.eq(0)
+
+        await withdrawHook
+          .connect(vault)
+          .hook(user.address, TEST_AMOUNT_BEFORE_FEE, TEST_AMOUNT_AFTER_FEE)
+
+        expect(await withdrawHook.getLastGlobalPeriodReset()).to.eq(
+          await getLastTimestamp(ethers.provider)
+        )
+      })
+
+      it('sets last global reset to current time if global period has passed (>)', async () => {
```

Comment:
nit:
```suggestion
      it('sets last global reset to current time if global period passed', async () => {
```
similarly for others